src/panels.cpp

```diff
@@ -1826,9 +1827,32 @@ const std::string panel_manager::get_current_layout_id() const
     return current_layout_id;
 }
 
+int panel_manager::get_width_right()
+{
+    if( get_option<std::string>( "SIDEBAR_POSITION" ) == "left" ) {
+        return width_left;
+    }
+    return width_right;
+}
+
+int panel_manager::get_width_left()
+{
+    if( get_option<std::string>( "SIDEBAR_POSITION" ) == "left" ) {
+        return width_right;
+    }
+    return width_left;
+}
```

Comment:
Looks like `get_width_right()` and `get_width_left()` are exactly the same, which probably means that you don't need both.

Reply:
They are not. They're reversed in which they provide, based on the state of sidebar.

Reply:
Ok, I really can't read